src/CLU/Microsoft.Azure.Commands.Compute/VirtualMachine/Config/AddAzureVMAdditionalUnattendContentCommand.cs

```diff
@@ -30,7 +30,7 @@ namespace Microsoft.Azure.Commands.Compute
         ProfileNouns.AdditionalUnattendContent),
     OutputType(
         typeof(PSVirtualMachine))]
-    [CliCommandAlias("vm additional unattend content add")]
+    [CliCommandAlias("vm config additionalunattendcontent add")]
```

Comment:
Why not ```additionalunattendcontent config```? If two aliases are in conflict, is it a build-time error, or run-time error?